tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.Windows_NT.x64.txt

```diff
@@ -0,0 +1,34 @@
+<Configuration> 
+ <TestRoot>..\..</TestRoot> 
+ <TargetArchitecture>x64</TargetArchitecture>
+ <ScriptRootDir>$(TestRoot)\Scripts</ScriptRootDir> 
+ <DebuggeeRootDir>$(TestRoot)\Debuggees</DebuggeeRootDir> 
+ <DebuggeeSourceRoot>$(DebuggeeRootDir)\Source</DebuggeeSourceRoot>
+ <RuntimeRoot>##Insert_Runtime_Root##</RuntimeRoot>
+ <NugetCacheRoot>##Insert_Nuget_Cache_Root##</NugetCacheRoot>
+ <TestProduct>ProjectK</TestProduct> 
+ <HostExe>$(RuntimeRoot)\corerun.exe</HostExe> 
+ <DbgShim>$(RuntimeRoot)\dbgshim.dll</DbgShim> 
+ <RuntimeSymbolsPath>$(RuntimeRoot)</RuntimeSymbolsPath> 
+ <DebuggeeBuildProcess>cli</DebuggeeBuildProcess> 
+ <CliVersion>1.0.0-preview1-002702</CliVersion> 
```

Comment:
This CliVersion is a lot older than the one in the debuggertests repo's config files (1.0.0-preview2.1-003155).  Is this used?